src/content/self-hosted/install/releases.mdx

```diff
@@ -6,17 +6,53 @@ id: releases
 
 ### Deprecation Notice
+
 - Support for Kubernetes [1.22](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md) has been deprecated and will be removed in the next release.
 
 ### Features
+
 - Kubernetes supported versions: [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md) ([upgrade guide](self-hosted/install/upgrade.mdx/#upgrade-to-kubernetes-125)), [1.24](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md) and [1.23](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md).
 - Support for custom [labels and annotations](self-hosted/administration/configuration.mdx#namespace) in ingresses managed by Okteto.
+- Upgraded the Okteto CLI version to [2.15.2](https://github.com/okteto/okteto/releases/tag/2.15.2)
+  - Okteto CLI 2.15.2 enables the optimisation removed at Okteto CLI 2.14.3. Now we calculate a hash for a given commit with the info around the build.
+- Oketo Self-Hosted will now show the [installation setup guide](administration/dashboard/#installation) by default
+- External resources can now be destroyed from the Okteto UI
+- External resources now supports [additional icons](reference/manifest.mdx##icon-string-optional)
+
+### Improvements
+
+- All external services created or updated by Okteto will now have the `dev.okteto.com/namespace` label 
+- Improved the performance of the query for retrieving endpoints for a preview environment using the `dev.okteto.com/namespace` label
+- Support baggage header for istio divert driver
+- Remove the `email` field from the helm values file. It is is now provided by the license
+- Divert virtual service transformation is now done at the mutation webhook by adding the annotation to the virtualservices
+- The frontend app now uses proper browser history routing instead of hash routing
+- Catalog items are now sorted in the same way in both the admin dashboard and a developer's deploy/redeploy modal
+- Using personal repositories from Bitbucket is now possible as we've addressed an issue with restricted characters in repository names
+- Persistent spaces can no longer be put to sleep from the admin dashboard
+
+### Bugfixes
+
+- Custom endpoints annotation `dev.okteto.com/endpoints` is now mapped with the correct service of an ingress
+- Fixed text in the UI is garbled when the deployment fails to be destroyed
+- Fix preview delete confirm modal stating "Delete namespace"
+- Tabs in the admin panel are now appropriately responsive for smaller screens and resolutions. The last tabs will now wrap properly
+- When a license is not found, an error will now be logged
+- External Resources markdown code is now displayed as an inline item instead of a standalone row block  
+- Some translation extensions where breaking the UI, this has been fixed
```

Comment:
Some will still break it. Not all translation extension respect the `translate` attribute.
Not sure how to rephrase it tho.

Reply:
How about something like this? Not sure if we need to get into the details of _how_?
```suggestion
- Translation browser extensions causing blank screens due to modifying DOM
```